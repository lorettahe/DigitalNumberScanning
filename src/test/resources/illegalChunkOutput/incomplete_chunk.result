000000000
123456789