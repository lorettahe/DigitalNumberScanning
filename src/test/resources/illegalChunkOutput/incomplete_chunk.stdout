Error on chunk 2: this illegal chunk has 2 lines instead of 3 lines!